csrc/executor_kernel_arg.h

```diff
@@ -371,6 +371,8 @@ class TORCH_CUDA_CU_API KernelArgumentHolder {
 
   void push(const ArgAbstract* arg);
 
+  void erase(std::unique_ptr<ArgAbstract>);
```

Comment:
Why not just `void erase(size_t ind)`?

Reply:
because there may be duplicates in the vector.